### DISCUSSION TITLE
air sdk 33.1.1.731 unexpected failure:zip file closed error when aab compiling

### DISCUSSION THREAD

**Discussion by mehmury** [airsdk/Adobe-Runtime-Support#1534] [Q&A]
I am debugging one of my android mobile applications. Some features that work with as3corelib do not work when I try them on my android 10 device. It works in previous android versions.
With the latest beta version sdk 33.1.1.731, features are working in local tests with Adobe Animate. But in this sdk version, I get unexpected failure error during aab compiling. I downloaded the android requirements specified in the sdk release notes and separate java 17.0.2 and java 11.0.12. I tried it separately, it always gives the same error.
how can i get aab build from this version?
![err22](https://user-images.githubusercontent.com/51246471/148409742-5d013538-ec6e-4c23-ad6e-f30eea8b68f3.png)

ABOUT SAME BUG: #1516

**Comment by jimmymjing** [comment]
I'm getting the same error for aab packaging.
